modules/plugin/jdbc/jdbc-postgis/src/main/java/org/geotools/data/postgis/PostGISDialect.java

```diff
@@ -1175,5 +1191,17 @@ protected void addSupportedHints(Set<Hints.Key> hints) {
             hints.add(Hints.GEOMETRY_SIMPLIFICATION);
         }
     }
+
+    public static class XDate extends java.util.Date {
```

Comment:
If this is a public class we will need javadocs, and probably an entry in the user guide.